### PR TITLE
Update Chinese translation and fix typo

### PR DESCRIPTION
**Pull request by vickyhella**
- Update Chinese translation and fix typo

### CODE REVIEW

**Comment by vickyhella** [comment]
@niusmallnan @kingsd041 Please help take a look at the translation update. Thanks!

File: shell/assets/translations/zh-hans.yaml [hunk 1]
```diff
@@ -1102,7 +1112,7 @@ cluster:
     advanced:
       label: 高级选项
       detail: 对节点注册方式的额外控制。每个注册的节点需要的值，通常都不相同。
-      nodeName: 节点名称
+      NodeName: 节点名称
```
Comment (niusmallnan) [inline shell/assets/translations/zh-hans.yaml:1115]: Hi @bisht-richa , do you know why `Nodename` is used as the key here? It uses an uppercase initial.

Maybe the en-us.yaml file needs an update and needs to see where it's referenced.

**Comment by kingsd041** [comment]
LGTM

File: shell/assets/translations/zh-hans.yaml [hunk 2]
```diff
@@ -4986,47 +4996,80 @@ storageClass:
...
+  global-Default: 全局默认
   host: 主机
+  hostIp: 主机 IP
+  holder: 持有者
+  hpaReference: 工作负载
   health: 健康
+  handSize: Hand Size
+  ipAddress: IP 地址
   id: ID
+  iP: IP
```
Comment (kingsd041) [inline shell/assets/translations/zh-hans.yaml:5072]: Hi @bisht-richa, Similar to `Nodename` above, it seems strange to use `iP` as the primary key, we usually use `ip` or `IP`

**Comment by nwmac** [comment]
@bisht-richa can you review please

File: shell/assets/translations/zh-hans.yaml (hunk 2 above)
Comment (bisht-richa) [reply shell/assets/translations/zh-hans.yaml:5072]: These are used in sortable tableHeaders.

**Review by bisht-richa — APPROVED** [review]
LGTM

**Pull request merged**